### PR TITLE
Register new package SLHA.jl v0.0.1

### PR DESCRIPTION
**Pull request by attobot**
Repository: [freeboson/SLHA.jl](https://github.com/freeboson/SLHA.jl)
Release: [v0.0.1](https://github.com/freeboson/SLHA.jl/releases/tag/v0.0.1)
Travis: [Travis Build Status](https://travis-ci.org/freeboson/SLHA.jl/builds/209367364)
cc: @freeboson

### CODE REVIEW

File: SLHA/versions/0.0.1/requires [hunk 1]
```diff
@@ -0,0 +1 @@
+julia 0.5-
```
Comment (tkelman) [inline SLHA/versions/0.0.1/requires]: do you need to support prereleases of 0.5?

Comment (freeboson) [inline SLHA/versions/0.0.1/requires]: Not at all. I thought it would be better to be more inclusive.

Comment (tkelman) [inline SLHA/versions/0.0.1/requires]: `0.5-` includes the very first commit after release-0.4 branched, so it's better at this point to use releases as lower bounds wherever possible

**Comment by freeboson** [comment]
@tkelman i've changed `julia 0.5-` to `julia 0.5` in REQUIRES

**Comment by tkelman** [comment]
Is there some term that describes what SLHA is? It's a long name and probably not worth spelling out in full, but what is it? A data file format?

**Comment by freeboson** [comment]
@tkelman Supersymmetry Les Houches Accord (arxiv:0801.0045)[https://arxiv.org/abs/0801.0045]. Yeah, a file format.

**Comment by tkelman** [comment]
Would it be any more descriptive to call it SLHAFormat or something like that?

**Comment by freeboson** [comment]
Compare with http://skands.physics.monash.edu/slha/ — the other implementations are `SLHAea`, `SLHALib`, `pySLHA`, etc. I think the particle physics community would expect the name `SLHA.jl`.

**Pull request merged**